src/python_testing/matter_testing_infrastructure/chip/testing/pics.py

```diff
@@ -96,9 +81,32 @@ def read_pics_from_file(path: str) -> dict[str, bool]:
             with open(filename, 'r') as f:
                 contents = f.read()
                 pics_dict.update(parse_pics_xml(contents))
+                print("reading pics ****")
```

Comment:
This seems like a debug print. Replace with logging module or delete debug prints